src/OrchardCore.Modules/OrchardCore.ReCaptcha/Manifest.cs

```diff
@@ -0,0 +1,27 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "ReCaptcha",
+    Author = "The Orchard Team",
+    Website = "http://orchardproject.net",
+    Version = "2.0.0")]
+
+[assembly: Feature(
+    Id = "OrchardCore.ReCaptcha",
```

Comment:
As soon as a module has at least one feature (or more), the module itself is no more a feature itself. Then one "rule" is to set the 1st feature id to the module id (assembly name). This way, for all OrchardCore things as permissions, admin menu, startups, controllers ... you don't have to decorate the related classes (of this 1st feature) with the `[Feature]` attribute. Note: but you need it for other features to separate things.

But this is not necessary, e.g in the distributed module i didn't use the module id for the 1st feature, then i had to decorate all that need to be decorated as for other features. I did this because i don't really have a **main feature**.

We need to declare this 1st first feature and then others can depend on it. We could just rename it but as said we would need to add feature attributes. But i think it's good to have the same name as the module id when the 1st feature is the "main feature" of the module.

In fewer words, all seems ok for me ;)